takwimu/templates/takwimu/_includes/report/country_selector.html

```diff
@@ -1,13 +1,25 @@
 {% load staticfiles %}
 
+{% block content %}
+<script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
```

Comment:
Please remove this. It's already loaded in the footer.

Reply:
Sure thing